src/Microsoft.AspNetCore.Razor/Compilation/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -24,6 +24,37 @@ public class TagHelperDescriptor
             Enumerable.Empty<TagHelperRequiredAttributeDescriptor>();
 
         /// <summary>
+        /// Creates a new <see cref="TagHelperDescriptor"/>. 
+        /// </summary>
+        public TagHelperDescriptor()
+        {
+
+        }
+
+        /// <summary>
+        /// Creates a copy of the given <see cref="TagHelperDescriptor"/>. Note: this is not a deep copy.
```

Comment:
`Creates a shallow copy of the given <see cref="TagHelperDescriptor"/>.`

Nuke the `Note: this is not a deep copy.`